content/en/apps/_index.md

```diff
@@ -21,5 +21,5 @@ Within the documentation, you will find set-up guides, tips, and operational ins
 
 Explore:
 
-* [Bank in a Box](../en/apps/bankinabox/getting-started.md) - A complete retail banking concept that can be hosted on a Corda network.
-* [State reissuance](../en/apps/reissuance/state-reissuance.md) - A mechanism for breaking transaction backchains by reissuing a state with a guaranteed state replacement.
+* [Bank in a Box](apps/bankinabox/getting-started.md) - A complete retail banking concept that can be hosted on a Corda network.
+* [State reissuance](apps/reissuance/state-reissuance.md) - A mechanism for breaking transaction backchains by reissuing a state with a guaranteed state replacement.
```

Comment:
I think I specifically updated this to the original link setup and these links are currently working - have you tested this fix?